src/hooks/useSingleStageHandler.ts

```diff
@@ -1,20 +1,53 @@
-import { getTimeRemaining } from "./../utils/dateTimeUtils";
+import { useProvider } from "wagmi";
+import { useParams } from "react-router-dom";
 import { useState, useEffect } from "react";
-import { IStage } from "./../interfaces/IStage";
+import { IStageMetaData } from "./../interfaces/IStage";
```

Comment:
what is IStageMetaData? Is this the data from ipfs object?

Reply:
IStage is the one that we are saving on ipfs with limited data that is meant to be saved on ipfs. IStageMetaData is the data that contains the stage data from ipfs as well as the data from contract

Reply:
please rename to IStage and separate IStageIPFSData vs IStageContractData

Reply:
Done